pandunia-lekse.csv

```diff
@@ -2440,7 +2440,7 @@ sare|badan||fas:(sar), hin:सिर (sir), ara:(raʾs)|head|tête|cabeza|cabeç
 saroste|badan|||skull||cráneo (calavera)||||||||||||||pääkallo|czaszka|
 sate|||ara:(sāʿa), fas:(sâ'at), may:tur:saat, swa:saa|moment (while, short time)||momento (tiempo)|||سَاعَة‎|||||||saat|saa|saat||hetki (kotva)|moment|
 saturne||||Saturn|Saturne|Saturno|Saturno|||土星|土星||||||||||Saturn|
-saudar|||may:saudara, hin:सहोदर (sahodar), ben:সহোদর (sôhodôr)|sibling (brother or sister)||hermano o hermana|||||||||||||frato aŭ fratino|sisarus (veli tai sisko)|rodzeństwo, brat lub siostra|
+saudar (saudare)|||may:saudara, hin:सहोदर (sahodar), ben:সহোদর (sôhodôr)|sibling (brother or sister)||hermano o hermana|||||||||||||frato aŭ fratino|sisarus (veli tai sisko)|rodzeństwo, brat lub siostra|
```

Comment:
Use the word with -e only. This word has gotten too long, so we should find a shorter word to replace it.

Reply:
I agree. The most international word I could find was Chinese "同生" /tʰuŋ˧˥ʂəŋ˥/ (it looses its coda in Japanese, so we could borrow it as "tonxe" or something). If we want to avoid borrowing a compound, there's also Latin "german-", which is in Spanish as "herman-" and Portuguese as "irmã-". We could also always shorten सहोदर to "sodare".